Task2/style2.css

```diff
@@ -1,251 +1,246 @@
-  body {
-    padding: 0;
-    margin: 0;
-    font-family: 'Times New Roman', Georgia, serif;
-  }
-  
-  h1, h2, h3 {  
-    letter-spacing: 5px;
-    font-weight: normal;
-  }
-  
-  .container {
-    max-width: 1600px;
-    min-width: 500px;
-    margin: auto;
-    position: fixed;
-    width: 100%;
-    z-index: 1;
-    background-color: white;
-    box-shadow: 0px 0px 30px #8a8585;
-    padding: 8px 16px;
-  }
+body {
+  padding: 0;
+  margin: 0;
+  font-family: 'Times New Roman', Georgia, serif;
+}
+  
+h1, h2, h3 {  
+  letter-spacing: 5px;
+  font-weight: normal;
+}
+  
+.container {
+  max-width: 1600px;
+  min-width: 500px;
+  margin: auto;
+  position: fixed;
+  width: 100%;
```

Comment:
When we render the HTML on the browser you can see the header going out of the page, this is because you have set width: 100% without including padding on both sides(box-model) ie, you need to include padding on both sides while applying width.width: calc(100% - 32px);
16px + 16px = 32px; padding on both sides.